support/cas-server-support-oauth/src/main/java/org/apereo/cas/config/CasOAuthThrottleWebMvcConfigurer.java

```diff
@@ -0,0 +1,30 @@
+package org.apereo.cas.config;
+
+import org.apereo.cas.web.support.AuthenticationThrottlingExecutionPlan;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+
+import static org.apereo.cas.support.oauth.OAuth20Constants.BASE_OAUTH20_URL;
+
+/**
+ * This is {@link CasOAuthThrottleWebMvcConfigurer}.
+ *
+ * @author sbearcsiro
+ * @since 5.3.0
+ */
+@Configuration
+public class CasOAuthThrottleWebMvcConfigurer extends WebMvcConfigurerAdapter {
```

Comment:
Could you turn this into a bean inside the original configuration class, and pass what it needs as a parameter, autowired, etc?